source/appendices/e-audit-requirements-and-tests/05-tr-book-tests.rst

```diff
@@ -0,0 +1,145 @@
+
+
+E.5.2.2 Available as Both Book_Segments and/or Reference_Items and Whole Books and/or Reference_Works
+'''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''
+
+These tests apply where Books and/or Reference_Works are available as both whole Books and/or Reference_Works and as Book_Segments and/or Reference_Items.
+
+**Option 1**: Report provider offers Book_Segments and/or Reference_Items and Whole Books and/or Reference_Works under only one Access_Type.
+
+The auditor MUST request 
+
+* 50 Book_Segments and/or Reference_Items, 10 each from 5 different Books and/or Reference_Works
+* 20 Books and/or Reference_Works, noting the count of Book_Segments and/or Reference_Items from each Title.
+
+This MUST result in 50, plus the count of Book_Segments and/or Reference_Items from the Title downloads, of each of Total_Item_Investigations, Total_Item_Requests, Unique_Item_Investigations and Unique_Item_Requests, and 27 each of Unique_Title_Investigations and Unique_Title_Requests with the appropriate Access_Type.
+
+**Option 2**: Report provider offers Book_Segments and/or Reference_Items under two different Access_Types.
+
+The auditor MUST request 
+
+* 30 Book_Segments and/or Reference_Items, 10 each from 3 different Books and/or Reference_Works, for each Access_Type
+* 10 Books and/or Reference_Works, noting the count of Book_Segments and/or Reference_Items from each Title, for each Access_Type.
+
+This MUST result in 30, plus the count of Book_Segments and/or Reference_Items from the Title downloads, of each of Total_Item_Investigations, Total_Item_Requests, Unique_Item_Investigations and Unique_Item_Requests, and 13 each of Unique_Title_Investigations and Unique_Title_Requests, for each of the two Access_Types.
+
+The Access_Type combinations might be: Controlled plus Open, Controlled plus Free_To_Read, or Open plus Free_To_Read
+
+**Option 3**: Report provider offers Book_Segments and/or Reference_Items under all three Access_Types (Controlled, Open and Free_To_Read).
+
+The auditor MUST request 
+
+* 20 Book_Segments and/or Reference_Items, 10 each from 2 different Books and/or Reference_Works, for each Access_Type
+* 5 Books and/or Reference_Works, noting the count of Book_Segments and/or Reference_Items from each Title, for each Access_Type
+
+This MUST result in 20, plus the count of Book_Segments and/or Reference_Items from the Title downloads, of each of Total_Item_Investigations, Total_Item_Requests, Unique_Item_Investigations and Unique_Item_Requests, and 7 each of Unique_Title_Investigations and Unique_Title_Requests, for each of the Access_Types.
+
+
+E.5.2.3 Available as Whole Books and/or Reference_Works
+'''''''''''''''''''''''''''''''''''''''''''''''''''''''
+
+These tests apply where Books and/or Reference_Works are only available as single downloads (i.e. no Book_Segments and/or Reference_Items can be identified).
```

Comment:
This is the case "... as single downloads and no Book_Segments and/or Reference_Items can be identified". The case "... as single downloads and Book_Segments and/or Reference_Items can be identified" is still missing.